src/web/src/components/Posts/Post.tsx

```diff
@@ -92,17 +83,79 @@ const useStyles = makeStyles((theme: Theme) =>
       lineHeight: '1.00',
       fontSize: '1em',
     },
+    postInfo: {
+      color: 'white',
+      width: '200px',
+      float: 'right',
+      marginRight: '-24em',
+      top: '8em',
+      bottom: '100%',
+      [theme.breakpoints.down(1200)]: {
+        width: '100%',
+        height: '2%',
+        float: 'none',
+        top: '8.4em',
+      },
+    },
+    authorInfoContainer: {
+      display: 'flex',
+      flexDirection: 'column',
+      borderLeft: '2.5px solid #707070',
+      width: '100%',
+      paddingLeft: '2em',
+      height: '100%',
+      [theme.breakpoints.down(1200)]: {
+        flexDirection: 'row',
+        justifyContent: 'center',
+        border: 'none',
+      },
+    },
+    authorNameContainer: {
+      width: 'calc(100% - 2em)',
+      [theme.breakpoints.down(1200)]: {
+        width: 'auto',
+        padding: '0 3em 0 1em',
+      },
+    },
+    publishedDateContainer: {
+      [theme.breakpoints.down(1200)]: {
+        fontSize: '0.6em',
+      },
+    },
+    authorAvatarContainer: {
+      shapeOutside: 'circle(50%) border-box',
+      shapeMargin: '1rem',
+      borderRadius: '50%',
+      float: 'left',
+      paddingBottom: '1em',
+      [theme.breakpoints.down(1200)]: {
+        float: 'none',
+        paddingBottom: '0px',
+      },
+    },
+    circle: {
+      display: 'block',
+      borderRadius: '50%',
+      backgroundColor: '#121D59',
+      width: '8em',
+      height: '8em',
+      [theme.breakpoints.down(1200)]: {
+        margin: '0.5em 0',
+        width: '4em',
+        height: '4em',
+      },
+    },
   })
 );
 
 const formatPublishedDate = (dateString: string) => {
   const date: Date = new Date(dateString);
   const formatted = new Intl.DateTimeFormat('en-CA', {
-    month: 'long',
+    month: 'short',
     day: 'numeric',
     year: 'numeric',
   }).format(date);
-  return `Last Updated ${formatted}`;
+  return `${formatted}`;
```

Comment:
Formatted is already a string, you don't need to use a template literal.  I would:

```js
return new Intl.DateTimeFormat(...).format(date);
```